src/bundle/Resources/public/js/scripts/admin.notifications.js

```diff
@@ -0,0 +1,32 @@
+(function (global, doc, eZ, $) {
+    const notificationsContainer = doc.querySelector('.ez-notifications-container');
+    const notifications = JSON.parse(notificationsContainer.dataset.notifications);
+    const addNotification = ({ detail }) => {
+        const configKey = detail.label === 'danger' ? 'error' : detail.label;
```

Comment:
Why can't we have `error` label instead?

Reply:
Bootstrap uses `danger` for errors and we cannot change it now because of BC. But this reminded me I forgot to add `TODO` for 3.0.